libs/state/src/lib/cell-summary/cell-summary.state.ts

```diff
@@ -41,6 +50,12 @@ export class CellSummaryState {
     patchState({ aggregates });
   }
 
+  @Action(UpdateSummaries)
+  updateSummaries({ patchState }: Context, { summaries }: UpdateSummaries): void {
+    const aggregates = summaries.map(computeAggregate);
+    patchState({ aggregates });
```

Comment:
Shouldn't this set the summaries as well? You could even reuse the existing logic.
```
patchState({ summaries });
this.computeAggregates()
```